lola.html

```diff
@@ -515,7 +515,7 @@ <h4>Changing Object IDs</h4>
       <h4>Actor ID</h4>
 
       <p>
-        The actor ID of an object MAY change or be kept the same.  In the personal account migration use case, presumably old and new content should be harmonized, and the new actor “owns” the content so it SHOULD be the new actor.  However, if an implementation is using account portability functions for a different use case, such as merging individual contributions into a group blog, it MAY keep the original actor.  A destination server should attempt to provide appropriate GUI affordances to be able to tell whether to keep or rewrite the Actor property.  For example, an account page with text like “Move my content from old account” would indicate a wish to update the actor ID. 
+        The actor ID of an object MAY change or be kept the same.  In the personal account migration use case, presumably old and new content should be harmonized, and the new actor “owns” the content so it SHOULD be the new actor.  However, if an implementation is using account portability functions for a different use case, such as merging individual contributions into a group blog, it MAY keep the original actor.  Nomadic identities may also result in keeping the same actor.  A destination server should attempt to provide appropriate GUI affordances to be able to tell whether to keep or rewrite the Actor property.  For example, an account page with text like “Move my content from old account” would indicate a wish to update the actor ID. 
```

Comment:
i'm hoping to ship a PR very soon (maybe today!) outlining one form of this, if the evolving draft [here](https://codeberg.org/fediverse/fep/src/branch/main/fep/ef61/fep-ef61.md) isn't easy enough to imagine in an e2e flow.